src/account-info.js

```diff
@@ -23,6 +23,17 @@ function AccountInfo (object) {
   this._isMobileVerified = Boolean(object.sms_verified);
 
   this._currency = object.currency;
+
+  var notifications = {};
+  if (object.notifications_type) {
+    object.notifications_type.forEach(function (code) {
+      var type = Math.log2(code);
```

Comment:
Taken from here https://github.com/blockchain/My-Wallet-V3-Frontend/blob/master/assets/js/services/wallet.service.js#L127

@jtormey  told me that notifications are stored as a bitfield on the server, with each bit corresponding to a different notification type. `0000 0000` is no notifications, `0000 0001` is email notifications, and `0001 0001 `is SMS and email. Log base 2 returns simpler numbers (0, 2, 5) than base 10. I could be missing a few details but that seemed to be the rationale.